app/models/concerns/orchestration/tftp.rb

```diff
@@ -132,6 +140,9 @@ def validate_tftp
     return unless tftp? || tftp6?
     return unless host.operatingsystem
     pxe_kind = host.operatingsystem.pxe_loader_kind(host)
+    if pxe_kind.to_s == "PXEGrub2TargetOS"
+      pxe_kind = "PXEGrub2".to_sym
```

Comment:
I would suggest to have a mapping function somehwere so that no hard coding is necessary here,
Like
```
pxe_kind_mapping = {
  'PXEGrub2TargetOS' => 'PXEGrub'
}
```
What do you think about this @ekohl - Maybe you have an good idea where this mapping should be.

Reply:
Deprecated with our new approach